frontend/src/theme/primitives/breakpoints.ts

```diff
@@ -0,0 +1,6 @@
+export const breakpoints = {
+  sm: "576px",
+  md: "768px",
+  lg: "992px",
+  xl: "1200px",
```

Comment:
Not sure if intentional but we also have xxl (1400px) and xs (0)

Reply:
I'll adjust these when we need media queries in our CSS.